pkg/cert/request.go

```diff
@@ -65,8 +65,8 @@ func NewServerCertificateManager(kubeClient clientset.Interface, namespace, secr
 			// authenticate itself to a TLS client.
 			certificates.UsageServerAuth,
 		},
-		CertificateStore:      certificateStore,
-		CertificateExpiration: certificateExpiration,
+		CertificateStore:    certificateStore,
+		CertificateRotation: certificateRotation,
```

Comment:
This change to `pkg/cert/request.go` seems to be unrelated to updating log messages and client-go. Would you mind pulling this out into its own separate PR please?

Reply:
I ran into the same issue when updating client-go. This is related to the client-go change, the metric was replaced internally and we are forced to make the change here

Reply:
PR here https://github.com/kubernetes/kubernetes/pull/85874

Reply:
The gauge metric seems to have been removed in client-go 1.18. Thanks @mcristina422 for the PR link. I was looking for it but couldn't find it.

Reply:
In my WIP branch here https://github.com/aws/amazon-eks-pod-identity-webhook/compare/master...mcristina422:updateK8sio?expand=1 I also had to change the managers clientFn

Reply:
I believe that would be necessary when we move to client-go for k8s 1.19. 

For the histogram metric above, I need to change the metric name and the help text, since the metric tracks certificate lifetime instead of expiration seconds. I will update that. 

I wonder if we should also update the README on this change on the metric. It seems to impact those who may have set alarms on the "server_expiration_seconds" gauge metric. Please let me know if there are concerns on this @jaypipes @mcristina422 @josselin-c @jqmichael @micahhausler

Reply:
Updated

Reply:
Gotcha, thanks for the explanation @mcristina422 :)